.github/workflows/ci.yaml

```diff
@@ -223,6 +223,12 @@ jobs:
           ./actionlint -color -shellcheck= -ignore "set-output"
         shell: bash
 
+      - name: Check for unstaged files
+        run: |
+          rm -f ./actionlint ./typos
```

Comment:
The `typos` action we use doesn't let you modify the binary install directory to anything other than the current directory (Is there a good reason why?)